src/Components/Footer/Footer.tsx

```diff
@@ -0,0 +1,31 @@
+import { TodoListItem } from "../../Interfaces/TodoListItem";
+import { FooterDiv, DeleteAllTaskButton } from "./FooterStyle";
+
+interface FooterProps {
+  setTodoList: React.Dispatch<React.SetStateAction<TodoListItem[]>>;
+  pendingTaskCount: number;
+  todoList: TodoListItem[];
+}
+
+const Footer: React.FC<FooterProps> = ({
+  setTodoList,
+  pendingTaskCount,
+  todoList,
+}) => {
+  const deleteAllTask = () => {
+    setTodoList([]);
+  };
+  return (
+    <FooterDiv>
+      <span>You have {pendingTaskCount} pending Task</span>
+      <DeleteAllTaskButton
+        className={todoList.length > 0 ? "active" : ""}
+        onClick={deleteAllTask}
+      >
+        Clear all
```

Comment:
Please add these strings in constant file and then use it here

Reply:
Can You explain.....I did'nt understand